Source/Fuse.Nodes/Visual.Children.uno

```diff
@@ -36,7 +36,7 @@ namespace Fuse
 			}
 			return null;
 		}
-
+		
```

Comment:
Something came and went from that file during my branch. I'll remove the remnant whitespace.